### DISCUSSION TITLE
Quality criteria

### DISCUSSION THREAD

**Discussion by HGauza** [nfdi4plants/ARC-specfication#19] [Comments and Suggestions]
I´m just posting that to make sure it doesn´t get lost.
Quality control and quality metrics are also discussed in the SDC context (BioDATEN, MoMaF ...). Maybe, there are ideas that can be exchanged between DataPLANT and the SDCs.

**Comment by muehlhaus** [comment]
Yes, this could be a topic for the future. Please, give a presentation in TA2 and/or TA3 meetings...
This would be very interesting.

**Reply by HGauza** [reply]
There is not much there, yet. I presented some ideas on what criteria might be. The slides are here: https://unitc-my.sharepoint.com/:p:/g/personal/obxga01_cloud_uni-tuebingen_de/EcobYqRJX5dLkc_M5OZzI9MBUNza_0CCUfV8DH5dqK94SA?e=MyAlwi
